### PR TITLE
[ur] unify result type naming convension

### PR DESCRIPTION
**Pull request by veselypeta**
This Commit also removes duplicate enums:
- KERNEL_NAME
- OUT_OF_HOST_MEMORY
- INVALID_PROGRAM_EXECUTABLE

The Following enum was removed
- INVALID_COMMAND_LIST

### CODE REVIEW

File: scripts/core/enqueue.yml [hunk 1]
```diff
@@ -1044,8 +1044,8 @@ params:
...
-    - $X_RESULT_INVALID_EVENT
-    - $X_RESULT_INVALID_MEM_OBJECT
-    - $X_RESULT_OUT_OF_HOST_MEMORY
-    - $X_RESULT_OUT_OF_RESOURCES
\ No newline at end of file
+    - $X_RESULT_ERROR_INVALID_QUEUE
+    - $X_RESULT_ERROR_INVALID_EVENT
+    - $X_RESULT_ERROR_INVALID_MEM_OBJECT
+    - $X_RESULT_ERROR_OUT_OF_HOST_MEMORY
+    - $X_RESULT_ERROR_OUT_OF_RESOURCES
```
Comment (kbenzie) [inline scripts/core/enqueue.yml:1051]: Please add a trailing new line to the end of this file.

File: scripts/core/queue.yml [hunk 2]
```diff
@@ -225,5 +225,5 @@ params:
       name: hQueue
       desc: "[in] handle of the queue to be flushed."
 returns:
-    - $X_RESULT_INVALID_QUEUE
+    - $X_RESULT_ERROR_INVALID_QUEUE
     - $X_RESULT_ERROR_OUT_OF_HOST_MEMORY
```
Comment (kbenzie) [inline scripts/core/queue.yml:229]: Another trailing new line here please.

File: scripts/core/usm.yml [hunk 3]
```diff
@@ -164,7 +164,7 @@ params:
...
 returns:
-    - $X_RESULT_INVALID_CONTEXT
-    - $X_RESULT_INVALID_VALUE
-    - $X_RESULT_INVALID_MEM_OBJECT
-    - $X_RESULT_OUT_OF_HOST_MEMORY
\ No newline at end of file
+    - $X_RESULT_ERROR_INVALID_CONTEXT
+    - $X_RESULT_ERROR_INVALID_VALUE
+    - $X_RESULT_ERROR_INVALID_MEM_OBJECT
+    - $X_RESULT_ERROR_OUT_OF_HOST_MEMORY
```
Comment (kbenzie) [inline scripts/core/usm.yml:170]: And another trailing new line please.

**Review by kbenzie — CHANGES_REQUESTED** [review]

**Review by FranklandJack — APPROVED** [review]
LGTM.

File: scripts/core/enqueue.yml (hunk 1 above)
Comment (veselypeta) [reply scripts/core/enqueue.yml]: :+1:

File: scripts/core/usm.yml (hunk 3 above)
Comment (veselypeta) [reply scripts/core/usm.yml]: :+1:

File: scripts/core/queue.yml (hunk 2 above)
Comment (veselypeta) [reply scripts/core/queue.yml]: :+1:

**Review by kbenzie — APPROVED** [review]

**Pull request merged**